applications_tests/lethe-particles/particle_solid_surface_CPES_double_edge_contact.prm

```diff
@@ -0,0 +1,138 @@
+# Test description:
+# In this test, a single particle is bouncing on a solid-surface made of two triangles that are coplanar (CP) and edge sharing (ES).
```

Comment:
License text indication for the prm file is missing. I know this is annoying but every file must have the license file ideally.